addons/web/models/models.py

```diff
@@ -3,13 +3,20 @@
 import pytz
 
 from odoo import _, api, fields, models
+from odoo.osv.expression import AND
 from odoo.tools import lazy
+from odoo.exceptions import UserError
+
+CATEGORY_SUPPORTED_TYPES = ['many2one']
```

Comment:
looks confusing at the top, maybe inline it in the methods, this way we don't have to find a proper name ;-)